source/jormungandr/jormungandr/planner.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         self.depth = depth
         self.isochrone_center = isochrone_center
         self.sn_params = sn_params
+        self.current_datetime = current_datetime or None
```

Comment:
Minor: The default value of current_datetime is already 'None' in the function

Reply:
updated in [a27c278448aa83761579ca9b198d3b0ad5af0d94](https://github.com/CanalTP/navitia/pull/3194/commits/a27c278448aa83761579ca9b198d3b0ad5af0d94)